### ISSUE TITLE
Option --timeout not works on Policies

### ISSUE THREAD

**Issue by gslav** [firewalld/firewalld#1381]
### Description:
`--timeout` works on Zones (remove Service from Zone after timeout) but does not on Policies.

### Expectation:
If add some Service to Policy with option `--timeout` i expect the Service to be removed after specified timeout.

### Result:
If add some Service to Policy with option `--timeout`, the Service persists in the Policy after specified timeout.
Same problem is present for RichRules.

### Steps to reproduce problem:
```sh
## create simple policy "test"
firewall-cmd --permanent --new-policy test
firewall-cmd --reload
## temporary add service
firewall-cmd --policy test --add-service https --timeout 10s
## check created policy
firewall-cmd --info-policy test
## check policy after timeout
sleep 10
firewall-cmd --info-policy test

## remove policy after test
firewall-cmd --permanent --delete-policy test
firewall-cmd --reload
```

### Environment:
- firewalld 2.2.1 on Arch Linux (up-to-date on 2024-08-28) with nftables as backend
- firewalld 2.1.2 on openSUSE Tumbleweed (up-to-date on 2024-08-28) with nftables as backend
- firewalld 1.2.1 on Oracle Linux 9.2 with nftables as backend
- firewalld 1.1.1 on Ubuntu 22.04.4 with nftables as backend

**Comment by erig0** [comment]
Indeed. See also RHEL bug: https://issues.redhat.com/browse/RHEL-5795
